cmd/server.go

```diff
@@ -61,25 +61,13 @@ func sleep() {
 
 func serverRun(cmd *cobra.Command, args []string) {
 	var client *pingdom.Client
+
 	flag.Parse()
 
-	if len(cmd.Flags().Args()) == 3 {
-		client = pingdom.NewClient(
-			flag.Arg(1),
-			flag.Arg(2),
-			flag.Arg(3),
-		)
-	} else if len(cmd.Flags().Args()) == 4 {
-		client = pingdom.NewMultiUserClient(
-			flag.Arg(1),
-			flag.Arg(2),
-			flag.Arg(3),
-			flag.Arg(4),
-		)
-	} else {
-		cmd.Help()
-		os.Exit(1)
-	}
+	client, _ = pingdom.NewClientWithConfig(
+		pingdom.ClientConfig{
+			APIToken: flag.Arg(3),
```

Comment:
I suppose the `flag.Arg(3)` is the API key. If the username and password are useless now, I would suggest to move the API key to the first argument position (and thus, remove the other arguments):

```golang
APIToken: flag.Arg(1),
```

Reply:
yes you're right